Integrations/integration-SCADAfence_CNM.yml

```diff
@@ -267,18 +272,85 @@ script:
         if asset_details:
             api_suffix = "/asset/traffic"
             return call_api('GET', api_suffix, asset_details)
-        raise "Invalid call for asset map (missing parameters)"
+        raise Exception("Invalid call for asset map (missing parameters)")
+
+    def dest_endpoint(ep):
+        return {
+                'ip': ep['dest_ip'],
+                'mac': ep['dest_mac'],
+                'hostname': ep['dest_hostname'],
+                'port': ep['dest_port'],
+                'proto': ep['proto'],
+                'traffic': ep['traffic']
+            }
+
+    def src_endpoint(ep):
+        return {
+                'ip': ep['src_ip'],
+                'mac': ep['src_mac'],
+                'hostname': ep['src_hostname'],
+                'port': ep['src_port'],
+                'proto': ep['proto'],
+                'traffic': ep['traffic']
+            }
+
+    def get_endpoint_data(data):
+        ret = []
+        for ep in data:
+            if 'ipAddress' in demisto.args().keys():
+                if demisto.args()['ipAddress'] == ep['src_ip']:
+                    ret.append(dest_endpoint(ep))
+                else:
+                    ret.append(src_endpoint(ep))
+
+            elif 'macAddress' in demisto.args().keys():
+                if demisto.args()['macAddress'] == ep['src_mac']:
+                    ret.append(dest_endpoint(ep))
+                else:
+                    ret.append(src_endpoint(ep))
+
+            else:
+                if demisto.args()['hostName'] == ep['src_hostname']:
+                    ret.append(dest_endpoint(ep))
+                else:
+                    ret.append(src_endpoint(ep))
+        return ret
 
     # The command demisto.command() holds the command sent from the user.
     if demisto.command() == 'test-module':
         # This is the call made when pressing the integration test button.
-        get_alerts('Critical', None)
+        get_alerts('Critical', None, None)
         demisto.results('ok')
         sys.exit(0)
 
-    elif demisto.command() == 'scadafence-getAlerts':
+    elif demisto.command() == 'scadafence-createAlert':
+
+        ip = demisto.args()['ipAddress']
+        severity = demisto.args()['severity']
+        description = demisto.args()['description']
+        active = demisto.args()['alertIsActive']
+        remediation = demisto.args()['remediationText']
+
+        api_suffix = "/alert"
+
+        status = call_api('POST', api_suffix, {
+            'ip': ip,
+            'severity': severity,
+            'details': description,
+            'active': active,
+            'remediation': remediation
+        })
+
+        demisto.results(status)
```

Comment:
Please improve the output message and add context outputs, so they can be used in a playbook.